service/src/main/java/com/myhome/controllers/HouseController.java

```diff
@@ -76,6 +85,20 @@
         .orElse(ResponseEntity.notFound().build());
   }
 
+  @Override public ResponseEntity<ListHouseHistoryResponse> getHouseHistory(String houseId,
+      @Valid String memberId) {
+    if (!houseService.getHouseHistory(memberId,houseId).isPresent()) {
+      return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new ListHouseHistoryResponse());
+    }
+    List<HouseHistory> houseHistoryList = houseService.getHouseHistory(memberId,houseId).get();
+    List<HouseHistoryDto> houseHistoryDtoList = houseHistoryList.stream()
+        .map(houseHistoryMapper::HouseHistoryToHouseHistoryDto)
+        .collect(Collectors.toList());
+    ListHouseHistoryResponse listHouseHistoryResponse = new ListHouseHistoryResponse().histories(houseHistoryDtoList);
+    return ResponseEntity.status(HttpStatus.OK).body(listHouseHistoryResponse);
```

Comment:
`ResponseEntity.status(HttpStatus.OK).body(listHouseHistoryResponse)`

can be simplified to:
`ResponseEntity.ok(listHouseHistoryResponse)`